script/Arcueid/Day 8/08_00_ARC08_8.txt

```diff
@@ -385,7 +385,7 @@ How do I know? It's simple.
 [sha:3dfdd0b667476035780f69e489a077676d97ed54]{
 -- Page 505, Offset 21024.
 -- コイツの狙っている個所は俺の体の“線”だった。
-Every attack of his has targeted one of the 'lines' on my body.
+Every attack of their has targeted one of the 'lines' on my body.
```

Comment:
Well the previous lines have it "their", I'll revert this one if you're aware of it.

Reply:
in this pattern, they should all be "every x of theirs"

Reply:
Well, it's a mix. I have
> -- **相手**の<挙|ふ><動|り>が読みやすいのか。
> Maybe **their** moves are just that easy to read?

but also 
> -- 何故なら狙われている個所が判る。[No subject lol go work it out]
> That's because I know what **he's** aiming for now.

and

> この**男**が防いでいる>    ということ。
> This **guy** is doing all he can to fend me off.

so it's a bit all over the place, but then so is the jp! Something like コイツ can be rendered either way, I dunno if we have an official policy but I think we have tended to gender it when it's known.

Reply:
since it's all over the place, there's a line where shiki says 

> 俺を襲ってきた人影……あれは男か The figure that attacked me...
 I guess it was a man...

we can call him a male right after that and leave the previous ones ambiguous

Reply:
Works for me